.editorconfig

```diff
@@ -0,0 +1,31 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+max_line_length = 120
+ij_visual_guides = 100,120,150
+
+[{go.mod,go.sum,*.go}]
+insert_final_newline = true
+indent_size = tab
+indent_style = tab
+tab_width = 2
```

Comment:
```suggestion
[{go.mod,go.sum,*.go}]
max_line_length = off # disabled because golangci-lint does it in a better way
insert_final_newline = true
indent_size = tab
indent_style = tab
tab_width = 2
```

Reply:
I wouldn't want to disable the line width limit here, as it will affect not only the linter, but also the code editor, which will also stop taking line length into account.

Reply:
Oh that's right. I assume everything is fine as golangci-lint and this setting won't conflict.

But a `//nolint` might have to be doubled by a `editorcheck-disable-line`

Reply:
> But a //nolint might have to be doubled by a editorcheck-disable-line

So far the problem was in the test file - I disabled the entire linter for editorconfig there. But if someone has to duplicate disabling checks somewhere else, then we can think about it and, perhaps, return to this proposal...